pkg/operator/ceph/csi/csi.go

```diff
@@ -79,6 +79,20 @@ func (r *ReconcileCSI) setParams() error {
 		CSIParam.ForceCephFSKernelClient = "true"
 	}
 
+	strval := k8sutil.GetValue(r.opConfig.Parameters, "ROOK_REVISION_HISTORY_LIMIT", "")
+	// var numval int32
+	var limit int32
+	if strval == "" {
+		limit = int32(10)
+	} else {
+		numval, err := strconv.ParseInt(strval, 10, 32)
+		if err != nil {
+			numval = int64(10)
+		}
+		limit = int32(numval)
+	}
+	CSIParam.RevisionHistoryLimit = limit
```

Comment:
For the CSI driver we are parsing it differently. Can we also set it to a nullable value, and nil by default? Even better if we could have a common method to parse the value instead of implementing in two places.

Reply:
done now. Avoided code duplication by using `SetRevisionHistoryLimit()` and `RevisioHistoryLimit()`in the csi part.